### PR TITLE
New version of how to get the HAR from Firefox

### PR DESCRIPTION
**Pull request by soulgalore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@sitespeed.io/geckodriver": "0.32.2",
     "@sitespeed.io/throttle": "5.0.0",
     "@sitespeed.io/tracium": "0.3.3",
+    "bidi-har-export": "firefox-devtools/bidi-har-export#b92585f39b6640ec93e57b0395b4dc22c2bde666",
```
Comment (juliandescottes) [inline package.json:15]: @soulgalore can you try with 55179f52bb77ea103569bb19ededef4c78d84048
```suggestion
    "bidi-har-export": "firefox-devtools/bidi-har-export#55179f52bb77ea103569bb19ededef4c78d84048",
```

**Comment by soulgalore** [comment]
@juliandescottes it seems that NodeJS 14 has problem accessing the repo?

**Comment by juliandescottes** [comment]
Thanks for the ping.

I can see threads about similar issue, but fixing it seems quite time consuming.
I'll see if I can get an agreement to publish that to npm, I think that would be the most efficient way to proceed here.

File: lib/firefox/webdriver/firefox.js [hunk 2]
```diff
@@ -3,15 +3,16 @@ import { promisify } from 'node:util';
 import { join } from 'node:path';
 import intel from 'intel';
 import get from 'lodash.get';
+import { adapters } from 'bidi-har-export';
```
Comment (juliandescottes) [inline lib/firefox/webdriver/firefox.js:6]:
```suggestion
import { adapters } from 'ff-test-bidi-har-export';
```

File: package.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
     "@sitespeed.io/geckodriver": "0.32.2",
     "@sitespeed.io/throttle": "5.0.0",
     "@sitespeed.io/tracium": "0.3.3",
+    "bidi-har-export": "firefox-devtools/bidi-har-export#55179f52bb77ea103569bb19ededef4c78d84048",
```
Comment (juliandescottes) [inline package.json:15]:
```suggestion
    "ff-test-bidi-har-export": "0.0.5",
```

**Review by juliandescottes — COMMENTED** [review]
We published a first version on npm, can you update and see if that works?

**Comment by soulgalore** [comment]
Thank you @juliandescottes !

**Comment by soulgalore** [comment]
Np I fixed that now.

**Comment by juliandescottes** [comment]
Thank you for following up here! Happy to take a look if there is any problem, don't hesitate to ping me.

**Pull request merged**